Mathlib/Data/Equiv/Functor.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Simon Hudon, Scott Morrison
+-/
+import Mathlib.Data.Equiv.Basic
+
+/-!
+
+# Functors can be applied to `equiv`s.
+
+```
+def functor.map_equiv (f : Type u → Type v) [functor f] [is_lawful_functor f] :
+  α ≃ β → f α ≃ f β
+```
+
+-/
+
+universes u v w
+
+variable {α β : Type u}
```

Comment:
```suggestion
variable {α β : Type u}
```

Also types that are lower-case Greek letters (and implicit) can typically be left out.

Reply:
What is the suggestion? It looks like you didn't change anything

Reply:
Sorry, the suggestion is to leave them out.

Reply:
Done. I was aware they could be left out, I just wasn't aware that it was what one was expected to do (it kind of feels counterintuitive to me)